docs/index.yml

```diff
@@ -261,21 +259,36 @@ additionalContent:
           - url: spark/index.yml
             text: .NET for Apache Spark
       # Card
+      - title: Game development
+        links:
+          - url: https://visualstudio.microsoft.com/vs/features/game-development/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link
+            text: Game development with Visual Studio
+          - url: https://docs.cryengine.com/display/CEPROG/C%23+Programming
+            text: Learn how to use CRYENGINE to build games with C#
+          - url: https://docs.monogame.net/?page=main
+            text: Build games with C# using the MonoGame library
+          - url: https://docs.unity3d.com/Manual/index.html
+            text: Learn how to use Unity to build 2D and 3D games with C#
+      # Card
       - title: Internet of things (IoT)
         links:
-          - url: ./iot/index.yml
+          - url: iot/index.yml
             text: .NET IoT Libraries
-          - url: ./iot/quickstarts/sensehat.md
+          - url: iot/quickstarts/sensehat.md
             text: Get started in 5 minutes
-          - url: ./iot/tutorials/blink-led.md
+          - url: iot/tutorials/blink-led.md
             text: Blink an LED
           - url: https://aka.ms/IoTNet101
             text: .NET IoT 101 video series
 
     - title: API and language reference # < 60 chars (optional)
       summary: Search the .NET API and language reference documentation. # < 160 chars (optional)
       items:
-        # Card
+      # Card
+      - title: ".NET API reference"
+        summary: API reference documentation for .NET
+        url: ../api/index.md?view=net-5.0
```

Comment:
```suggestion
        url: ../api/index.md?view=net-5.0&preserve-view=true
```

Reply:
For this one, I don't think we want to preserve the view. That way, the default (highest value) moniker will be used.